webapp/src/app/TeamChooser.jsx

```diff
@@ -84,9 +88,9 @@ const TeamChooser = ({players, movePlayer}) => {
 
     return (
         <Grid container spacing={2} justify="center" alignItems="center" className={classes.root}>
-            <Grid item>{customList(red, classes.red)}</Grid>
-            <Grid item>{customList(unassigned, classes.unassigned)}</Grid>
-            <Grid item>{customList(blue, classes.blue)}</Grid>
+            <Grid item>{customList("RED", red, classes.red)}</Grid>
```

Comment:
w wiekszosci miejsc obslugujmy nazwy druzyn lower casem

Reply:
wiec pewnie albo chciales lower albo odnosic sie do Team.RED

Reply:
To w sumie tylko string do wyświetlania w nagłówku. Ja bym tymczasowo zostawił 🤷‍♂️